dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerControllerSupport.java

```diff
@@ -30,5 +30,5 @@
 
 public class TrackerControllerSupport
 {
-    public static final String RESOURCE_PATH = "/tracker";
+    public static final String RESOURCE_PATH = "tracker";
```

Comment:
Are you sure this is correct?
We are using the `/` for any other controller

Reply:
Spring request mapping will add a / if needed. I'll will double check everything works though